hack/setup-kind-in-ci.sh

```diff
@@ -91,10 +91,6 @@ data:
       addresses: ['${first_ip}-${last_ip}']
 LBCONFIGMAP
 
-# if [ -n "${ISTIO_VERSION}" ]; then
-# DOWNLOAD_ISTIO_VERSION_ARG="--istio-version ${ISTIO_VERSION}"
-# fi
-
```

Comment:
if you aren't going to be able to tell this script to run against a specific Istio version, then remove all the obsolete code that deals with passing in the `-iv` command line option.

Although, I think you should keep this. Some branches might not support running against the latest version of Istio.

Reply:
I was just thinking about this. Since this script will run against backported PRs and releases, it'll need to support running the correct version of istio/kubernetes for the version of kiali we are using. Kiali provides the compatibility matrix after it starts up but in this case we need it before Kiali is running to determine which version of istio/kube to deploy. Probably can just keep a short list of the maistra supported releases to pin to a specific version of istio/kube and then use the latest istio/kube for all the rest?